docs/features/boundary-length-modifier/blm-calibration.hld.md

```diff
@@ -0,0 +1,83 @@
+# BLM Calibration - high-level design
+
+This document describes the current high-level overview of the Marxan Cloud
+platform user features related to calibration of the BLM (boundary length
+modifier) for a Marxan scenario.
+
+## Aim
+
+The BLM value chosen for a Marxan analysis can greatly affect the spatial
+distribution/clumping of planning units selected by the solver.
+
+The BLM calibration process designed for the Marxan Cloud platform will allow
+users to have an “at a glance” overview of the spatial distribution of
+solutions, given a discrete set of BLM values.
+
+The overview is presented to the user via a thumbnail snapshot of the solutions
+for each BLM value being tested, alongside the cost of the best solution for
+each of them.
+
+Once the calibration results are presented to the user, they may then choose to:
+
+- Select one of the calibrated BLM values, to implement in the scenario (this
+  will set the BLM property for the scenario)
+- Change one or more 6 values of the previous calibration run, and re-run the
+  calibration
+- Input their own BLM value (without using the platform's calibration feature)
+
+## Overview of the process
+
+For an overview of the process, see `data/notebooks/Lab/marxan_blm.ipynb`.
+
+The gist: this is done by running the Marxan solver on the same input files that
+would be used for the actual user-triggered Marxan run, once per BLM value in
+the given range, with a limited number of iterations per run (10) and only
+varying the BLM value in the input.dat file, all the rest being kept equal
+across each of the runs.
+
+## Requirements
+
+- Users should be able to set their BLM values.
+
+The set should be limited to a small cardinality: 6 values for the initial
+implementation, but leaving space for adjustments in the future (i.e. it is ok
+to hardcode the size of the set, but changing it in the future should not break
+other assumptions).
```

Comment:
So the initial run should already be N-setups (each for M runs) in parallel for given scenario and those values can be manually modified by the user? (i.e. N inputs presented with pre-calculated values with formula below)

Reply:
Also:
*  which scores are presented to user once the run above finishes?
* what about metadata setting of BLM value?

Reply:
it may eventually become a further option in the analysis tab, that the users can use to explore different settings before running the scenario (https://projects.invisionapp.com/share/G211WEUZFST9#/screens/459986030_Marxan_09a).

Once happy with BLM calibration results, users will typically run Marxan, either after setting BLM from one of the suggestions from the calibration, or if they want to ignore these, using either the default or a value they choose (they may have this from analysis they did outside of the platform).